src/borg/helpers.py

```diff
@@ -143,10 +143,15 @@ def __delitem__(self, name):
         del self._archives[name]
 
     def list(self, sort_by=None, reverse=False):
-        # inexpensive Archive.list_archives replacement if we just need .name, .id, .ts
-        archives = self.values()  # [self[name] for name in self]
+        """ Inexpensive Archive.list_archives replacement if we just need .name, .id, .ts
+
+            :rtype: A :class:`list` of :class:`borg.helpers.ArchiveInfo` instances
+        """
```

Comment:
please have a look at the usual docstring style and use the same. do a fixup commit.